packages/design-system/src/components/Table/Table.jsx

```diff
@@ -115,7 +123,7 @@ export class Table extends React.PureComponent {
       'aria-labelledby': this.captionID,
       'aria-live': 'polite',
       'aria-relevant': 'additions',
-      tabIndex: this.state.isTableScrollable ? '0' : null,
+      tabIndex: this.state.scrollActive ? '0' : null,
```

Comment:
@davidakennedy usually wouldnt we set the `tabIndex` to `-1` for enabling focus on a non focusable element?

Reply:
Looks like `tabIndex` of `0` is for keyboard focus - https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex